tubular/mixins.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+import pandas as pd
+
+
+class WeightColumnMixin:
+    def check_weights_column(X: pd.DataFrame, weights_column: str) -> None:
+        """Helper method for validating weights column in dataframe.
+
+        Args:
+        ----
+            X (pd.DataFrame): df containing weight column
+            weights_column (str): name of weight column
+
+        """
+        # check if given weight is in columns
+        if weights_column not in X.columns:
+            msg = f"weight col ({weights_column}) is not present in columns of data"
```

Comment:
Could we add 'self.classname(): ' to prefix the error msgs please?

Reply:
Good idea! Done here:  [d9ffd84](https://github.com/lvgig/tubular/pull/239/commits/d9ffd846e596da9a55e9ad39fa914c5b2b4ef14e)